packages/imperative/src/config/cmd/list/list.definition.ts

```diff
@@ -39,5 +39,27 @@ export const listDefinition: ICommandDefinition = {
                 "For example, specify in addition to '--locations' to get a list of config file paths only.",
             type: "boolean"
         }
+    ],
+    examples: [
+        {
+            description: "List property names for a specified config property.",
+            options: `"default"`
```

Comment:
Should this be plural "defaults" to match the property name in zowe.config.json?

Reply:
Right, I'll make it plural.